infra/aws/terraform/prow-build-cluster/resources/kubecost/prow-build-cluster-values.yaml

```diff
@@ -0,0 +1,51 @@
+## save this to values.yaml file and run
+## k create configmap kubecost-helm-values --from-file=values.yaml -n kubecost
+## on the prow-build-cluster
```

Comment:
Can we make this part of the `flux-update.bash` script?

Reply:
The problem here is that the values are different for canary and prod clusters. Otherwise, I could have just included them in the helmreleease file as well.

I can do it with a command like `kubectl cluster-info` (which reveals like `Kubernetes control plane is running at https://xxx.gr7.us-east-2.eks.amazonaws.com`) and use it with an if statement to create it. Wdyt?

Reply:
We have `PROW_ENV` variable in Makefile that can be `canary` or `prod`. We use that variable mainly for Terraform, but we can re-use it for other purposes as well.

What do you think about:

- Modifying this script to fail very early (at the beginning) if `PROW_ENV` variable is unset
- Modifying Makefile and `flux-update` target to pass `PROW_ENV` to the script (replace `./hack/flux-update.bash` with something like `PROW_ENV=$(PROW_ENV) ./hack/flux-update.bash`)
- Modifying this script to act based on the value of `PROW_ENV`

Reply:
Ok, that makes sense. I will implement it and update the PR accordingly.

Reply:
I changed it to use values file (instead of ConfigMap), and change the filename as `{canary,prod}-cluster-values.yaml`